src/lib/tzcode/CMakeLists.txt

```diff
@@ -1,2 +1,53 @@
-add_library(tzcode STATIC strftime.c strptime.c timelocal.c timezone.c)
+add_library(tzcode STATIC
+            strftime.c strptime.c timelocal.c localtime.c timezone.c)
 target_link_libraries(tzcode)
+
+set(TZCODE_PROJECT tzcode-github)
+set(TZCODE_BUILD_DIR ${CMAKE_CURRENT_BINARY_DIR})
+set(TZCODE_PREFIX_DIR tz)
+set(TZCODE_PREFIX ${TZCODE_BUILD_DIR}/${TZCODE_PREFIX_DIR})
+set(TZDATA_TARGETS main.zi)
+set(TZCODE_SRC ${PROJECT_SOURCE_DIR}/src/lib/tzcode)
+
+list(TRANSFORM TZDATA_TARGETS
+     PREPEND ${TZCODE_PREFIX}/src/${TZCODE_PROJECT}/
+     OUTPUT_VARIABLE TZDATA_FULL_TARGETS)
+
+ExternalProject_Add(${TZCODE_PROJECT}
+    GIT_REPOSITORY https://github.com/eggert/tz
```

Comment:
I wanted to use particular branch, to avoid using of moving target as master. And Paul Eggert is current (sole) maintainer of IANA timezones library.

I used Github link instead of https://data.iana.org/time-zones/releases/ because I do not know their retention policy (apparently they keep everything forever). Will change to using of tar.gz there to the particular branch.

Reply:
Changed to using tar.lz instead:
```diff
diff --git a/src/lib/tzcode/CMakeLists.txt b/src/lib/tzcode/CMakeLists.txt
index 31309b37b..72d7aef65 100644
--- a/src/lib/tzcode/CMakeLists.txt
+++ b/src/lib/tzcode/CMakeLists.txt
@@ -2,7 +2,7 @@ add_library(tzcode STATIC
             strftime.c strptime.c timelocal.c localtime.c timezone.c)
 target_link_libraries(tzcode)
 
-set(TZCODE_PROJECT tzcode-github)
+set(TZCODE_PROJECT tzcode-iana)
 set(TZCODE_BUILD_DIR ${CMAKE_CURRENT_BINARY_DIR})
 set(TZCODE_PREFIX_DIR tz)
 set(TZCODE_PREFIX ${TZCODE_BUILD_DIR}/${TZCODE_PREFIX_DIR})
@@ -14,10 +14,8 @@ list(TRANSFORM TZDATA_TARGETS
      OUTPUT_VARIABLE TZDATA_FULL_TARGETS)
 
 ExternalProject_Add(${TZCODE_PROJECT}
-    GIT_REPOSITORY https://github.com/eggert/tz
-    GIT_TAG 2022a
-    GIT_SHALLOW YES
-
+    URL https://data.iana.org/time-zones/releases/tzdb-2022a.tar.lz
+    URL_MD5 860c6f13b0b799e0d17ec16370f51957
     EXCLUDE_FROM_ALL 1
     INSTALL_DIR ${TZCODE_BUILD_DIR}/install
     PREFIX ${TZCODE_PREFIX_DIR}
```

Reply:
Decided to go with GH repo.